<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget_menu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <column>
     <property name="text">
      <string>功能菜单</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>室内定位</string>
     </property>
     <item>
      <property name="text">
       <string>实时定位</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>定位参数</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>查询统计</string>
     </property>
     <item>
      <property name="text">
       <string>定位查询</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>考勤记录</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>报警记录</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>基础设置</string>
     </property>
     <item>
      <property name="text">
       <string>部门管理</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>岗位管理</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>车辆类型</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>人员管理</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>基站管理</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>系统设置</string>
     </property>
     <item>
      <property name="text">
       <string>用户管理</string>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QTreeWidget" name="treeWidget_group">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <column>
     <property name="text">
      <string>人员类别(id)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>姓名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>标识</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>坐标</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>医生</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <item>
      <property name="text">
       <string>10001</string>
      </property>
      <property name="text">
       <string>火麟飞</string>
      </property>
      <property name="text">
       <string>红色</string>
      </property>
      <property name="text">
       <string>(103,340)</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>教师</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>学生</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>警察</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>显示</string>
    </property>
    <addaction name="action_grid"/>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_pause"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>列表查看</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查询</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>倍速</string>
     </property>
     <addaction name="action1_0x"/>
     <addaction name="action1_5x"/>
     <addaction name="action1_0x_2"/>
     <addaction name="action1_25x"/>
     <addaction name="action1_5x_2"/>
     <addaction name="action2_0x"/>
    </widget>
    <addaction name="action_id"/>
    <addaction name="action_locusplay"/>
    <addaction name="action_pause_2"/>
    <addaction name="menu_5"/>
    <addaction name="action_bujin"/>
    <addaction name="action_15"/>
    <addaction name="action_16"/>
    <addaction name="action_17"/>
    <addaction name="action_18"/>
    <addaction name="action_19"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>另存为</string>
     </property>
     <addaction name="action_22"/>
     <addaction name="action_23"/>
     <addaction name="action_26"/>
     <addaction name="action_27"/>
    </widget>
    <addaction name="action_20"/>
    <addaction name="menu_4"/>
    <addaction name="action_24"/>
    <addaction name="action_25"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_grid">
   <property name="text">
    <string>网格化</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>存储</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>更改比例</string>
   </property>
  </action>
  <action name="action_pause">
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>待设置</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>待设置</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>查看某一类别</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>查看某一个人</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>标签查看</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>待设置</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>待设置</string>
   </property>
  </action>
  <action name="action_id">
   <property name="text">
    <string>特定id</string>
   </property>
  </action>
  <action name="action_locusplay">
   <property name="text">
    <string>轨迹播放</string>
   </property>
  </action>
  <action name="action_pause_2">
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="action_bujin">
   <property name="text">
    <string>步进</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>特定时刻所有标签的位置示</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>某一时刻统计距离</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>某一个时刻统计列表</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>待设置</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>待设置</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_22">
   <property name="text">
    <string>根据时间段导出</string>
   </property>
  </action>
  <action name="action_23">
   <property name="text">
    <string>根据类别段导出</string>
   </property>
  </action>
  <action name="action_24">
   <property name="text">
    <string>待设置</string>
   </property>
  </action>
  <action name="action_25">
   <property name="text">
    <string>待设置</string>
   </property>
  </action>
  <action name="action_26">
   <property name="text">
    <string>待设置</string>
   </property>
  </action>
  <action name="action_27">
   <property name="text">
    <string>待设置</string>
   </property>
  </action>
  <action name="action1_0x">
   <property name="text">
    <string>0.5x</string>
   </property>
  </action>
  <action name="action1_5x">
   <property name="text">
    <string>0.75x</string>
   </property>
  </action>
  <action name="action1_0x_2">
   <property name="text">
    <string>1.0x</string>
   </property>
  </action>
  <action name="action1_25x">
   <property name="text">
    <string>1.25x</string>
   </property>
  </action>
  <action name="action1_5x_2">
   <property name="text">
    <string>1.5x</string>
   </property>
  </action>
  <action name="action2_0x">
   <property name="text">
    <string>2.0x</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>更改主题</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>待设置</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>待设置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
